engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
A couple of things about this:
1) I retracted this change: I'm dealing with it in PR #1565.
2) I misunderstood your suggestion @msteiger. You are suggesting to check explicitly for the extensions that bring specific opengl commands, not necessarily requiring entire opengl releases, right? A bit like we are doing already for frame buffer objects and float textures. 

It's not a bad idea theoretically, if I understood it right, at least from our point of view. Practically at this stage I'd find it tricky to come up retrospectively with a list of extensions used. Once a command is part of an opengl release we'd have to manually go back when it was introduced and find if it was part of a pre-existing extension. Perhaps it's a simple thing but right now I don't know how I'd do it efficiently. 

The other concern is how this would work from a user perspective. GPUs do not normally come with a big bright list of extensions supported. For example I just checked the [specs](http://www.geforce.co.uk/hardware/notebook-gpus/geforce-gtx-980m/specifications) for the GTX 980M and it just mentions OpenGL 4.1, additional extensions are not listed. So, from a user perspective a more detailed list of extensions/features used would not tell them in advance if Terasology would or would not run on their system.

Perhaps I'm still misinterpreting your suggestion...